### PR TITLE
Create h3lmcserver.json

### PR DESCRIPTION
**Pull request by gvalue-04**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [ ] You have completed your website. 
- [x] The website is reachable.  
- [ ] You're not using Vercel.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview
*insert minecraft world here*

### CODE REVIEW

**Comment by gvalue-04** [comment]
This is for a minecraft server, so it does not have an accessible website. You can however connect to the server using Minecraft Java Edition 1.12.2

File: domains/h3lmcserver.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Documentation for is-a-dev",
+  "repo": "https://github.com/is-a-dev/docs",
+  "owner": {
+    "username": "gvalue-04",
+    "email": "vaporvault04@gmail.com"
+  },
+  "record": {
+    "CNAME": "on-clan.gl.at.ply.gg:25565"
```
Comment (DEV-DIBSTER) [inline domains/h3lmcserver.json:9]: CNAME cannot have port number.

**Review by DEV-DIBSTER — DISMISSED** [review]

**Comment by EducatedSuddenBucket** [comment]
> on-clan.gl.at.ply.gg

hey you dont need cname with port , if you set your cname to on-clan.gl.at.ply.gg all ports on on-clan.gl.at.ply.gg will be accesible from your subdomain

**Comment by gvalue-04** [comment]
Updated

**Comment by gvalue-04** [comment]
the domain should be mc.h3l.is-a.dev

File: domains/mc.h3l.is-a.dev.json
Comment (CuteDog5695) [inline domains/mc.h3l.is-a.dev.json:1]: remove .is-a.dev from the name

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by gvalue-04** [comment]
Fixed

**Review by iostpa — APPROVED** [review]
Seems fine to me ig

**Pull request merged**